components/camel-test-spring/src/main/java/org/apache/camel/test/spring/CamelSpringBootExecutionListener.java

```diff
@@ -31,6 +33,7 @@ public void prepareTestInstance(TestContext testContext) throws Exception {
         LOG.info("@RunWith(CamelSpringBootJUnit4ClassRunner.class) preparing: {}", testContext.getTestClass());
 
         Class<?> testClass = testContext.getTestClass();
+        SpringCamelContext.setNoStart(true);
```

Comment:
Ah nice way to not start. Maybe add a little bit of code comment that this is on purpose and we do it to ensure the other stuff gets setup before camel is started.

I wonder if the regular camel-test-spring junit classes need anything like this, or do they already do something similar, in case they need to do some preparation and force camel to be started last. Just a thought.